### PR TITLE
[circle-mlir/models] Add Conv2d_F32_R4_p11_g2 test model

### PR DESCRIPTION
**Pull request by batcheu**
It adds the Conv2d test model with padding and group set.

ONE-DCO-1.0-Signed-off-by: Jonghwa Lee <jonghwa3.lee@samsung.com>

---

- issue : https://github.com/Samsung/ONE/issues/16394
- draft : #16393

### CODE REVIEW

File: circle-mlir/models/unit/Conv2d_F32_R4_p11_g32/__init__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        return self.op(input)
+
+    def onnx_opset_version(self):
+        # TODO set to appropriate value
+        return 14
+
+
+_model_ = net_Conv2d()
+
+_inputs_ = torch.randn(1, 256, 200, 320)
```
Comment (hseok-oh) [inline circle-mlir/models/unit/Conv2d_F32_R4_p11_g32/__init__.py]: Isn't it too big to test? Why don't you reduce input size?

Comment (batcheu) [reply circle-mlir/models/unit/Conv2d_F32_R4_p11_g32/__init__.py]: Yes it's a bit large but I want to use same arguments as the issued model.
Do you want me to reduce the size?

Comment (hseok-oh) [reply circle-mlir/models/unit/Conv2d_F32_R4_p11_g32/__init__.py]: > Do you want me to reduce the size?

Right. I want to reduce test time as possible
- For faster test on developer environment
- To reduce CI infra overhead

File: circle-mlir/models/unit/Conv2d_F32_R4_p11_g2/__init__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import torch
+
+
+# Generate Conv2d operator with Float32, Rank-4 with groups
+class net_Conv2d(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+        self.op = torch.nn.Conv2d(4, 4, 1, padding=(1, 1), groups=2)
```
Comment (batcheu) [inline circle-mlir/models/unit/Conv2d_F32_R4_p11_g2/__init__.py:8]: I reduced tensor size referring to @hseok-oh comment.

```bash
loc("/op/Conv/split_input"): error: 'Circle.split' op output #0 should be 'tensor<1x5x5x2xf32>' instead got 'tensor<1x3x3x2xf32>
```

**Comment by batcheu** [comment]
@shs-park , @hseok-oh 
I updated the commit, PTAL ;)

**Review by shs-park — APPROVED** [review]
👍
